src/components/panel/panel.tsx

```diff
@@ -196,7 +223,7 @@ export class Panel implements ConditionalSlotComponent, InteractiveComponent {
   @Event() calcitePanelDismiss: EventEmitter;
 
   /**
-   * Fires when the close button is clicked.
+   * Fires when there is a change in dismissed prop value .
```

Comment:
Nitpick:
We try not to abbreviate words in the doc. Also property/slot/event/css-var names should be in back ticks. So:

> Fires when there is a change to the \`dismissed\` property value.